### ISSUE TITLE
Bug: Unable to set deep objects to models without mutators

### ISSUE THREAD

**Issue by askmatey** [asciidisco/Backbone.Mutators#34]
You can not set deep objects to models without any `mutators`.

Example (*data recieved from a form*):
```
formData = {
  name: "Batman",
  car: "Batmobile",
  engine: "Jet Turbine",
  enemies: ["Godzilla", "Rodan", "Mothra", "Gorosaurus"],
  location: {
    atHome: false,
    inCave: true
  }
  ...
};

model.set(formData);
```

Error message:
```
Line 138: TypeError: can't convert key to string
```

The only solution is to change/undeclare the `mutators:{}` property to `mutators: undefined` in the model. Not that clean.

**Comment by asciidisco** [comment]
Merged & released in v 0.4.5

**Comment by askmatey** [comment]
+1
